nav2_lifecycle_manager/src/lifecycle_manager_client.cpp

```diff
@@ -112,7 +118,7 @@ LifecycleManagerClient::callService(uint8_t command, const std::chrono::nanoseco
     node_->get_logger(), "Waiting for the %s service...",
     manage_service_name_.c_str());
 
-  while (!manager_client_->wait_for_service(std::chrono::seconds(1))) {
+  while (!manager_client_->wait_for_service(timeout)) {
```

Comment:
Revert this back to 1s, this is specific to the lifecycle management and we want a generous timeout